httpclient5/src/main/java/org/apache/hc/client5/http/impl/classic/ContentCompressionExec.java

```diff
@@ -107,6 +125,7 @@ public ContentCompressionExec(final boolean ignoreUnknown) {
      * <ul>
      * <li>gzip - see {@link java.util.zip.GZIPInputStream}</li>
      * <li>deflate - see {@link org.apache.hc.client5.http.entity.DeflateInputStream}</li>
+     * <li>brotli - see {@link org.brotli.dec.BrotliInputStream}</li>
```

Comment:
The key says `br`, but this `brotli`. Intentional?

Reply:
Yes, br is abbreviations of brotli. When in HTTP header Accept-Encoding, br is used.